packages/create-cloudflare/templates/hello-world-durable-object/c3.ts

```diff
@@ -2,6 +2,8 @@ export default {
 	configVersion: 1,
 	id: "hello-world-durable-object",
 	displayName: "Hello World Worker Using Durable Objects",
+	description:
```

Comment:
The template seems a little more bare bones than this to me. Where are the multiplayer application parts in the template?